prime-router/src/main/kotlin/Receiver.kt

```diff
@@ -87,7 +87,8 @@ open class Receiver(
         jurisdictionalFilter: ReportStreamFilter = emptyList(),
         qualityFilter: ReportStreamFilter = emptyList(),
         routingFilter: ReportStreamFilter = emptyList(),
-        processingModeFilter: ReportStreamFilter = emptyList()
+        processingModeFilter: ReportStreamFilter = emptyList(),
+        reverseTheQualityFilter: Boolean = false
```

Comment:
Update the organizations.yml to add `reverseTheQualityFilter: false` to the "full-elr-hl7" receiver